packages/terra-functional-testing/src/commands/hide-input-caret.js

```diff
@@ -0,0 +1,20 @@
+const Logger = require('../logger/logger');
+
+/**
+ * Hides the blinking input caret that appears within editable text areas to prevent inconsistent test failures.
+ *
+ * @param {string} selector The selector for the element to hide the caret of
+ */
+const hideInputCaret = (selector) => {
+  try {
+    global.browser.execute(`document.querySelector("${selector.replace(/"/g, '\\"')}").style.caretColor = "transparent";`);
```

Comment:
I wonder if we should co-locate any of these browser.execute commands since they are theoretically the only things we need to transpile, i think?

Reply:
I think this is safe to not transpile, but now I'm second guessing that. After the recent changes to compile the visual regression service I'm starting to wonder if we should just compile everything so we can use ES6 and avoid mix and matching transpiled/non-transpiled

Reply:
I'm not against that, considering my almost oops with the optional chaining bug in terra-toolkit-boneyard

Reply:
i forget, did we initially choose to mix and match for efficiency gains?

Reply:
The visual regression service had code that also needs to be transpiled

Reply:
There are a bunch of scripts co-located for the visual-regression service here: https://github.com/cerner/terra-toolkit/tree/main/packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts

Might not be bad to follow this pattern later on. We can also use old-school func syntax for this to run in IE
```js
global.browser.execute(function () {
  document.querySelector("${selector.replace(/"/g, '\\"')}").style.caretColor = "transparent";`
})
```